Scripts/CAN/BPS.csv

```diff
@@ -1,43 +1,48 @@
-﻿Data,CAN ID,Length(bytes),Index Used,IN/OUT,Description
+﻿Data,CAN ID,Length(bytes),Index Used,IN/OUT,Description
 BPS Trip,0x002,1,0,,"Precision: 1 bit
 Data: 1 - bps tripped, something is wrong
 0 - bps has not tripped. Could either be battery pack is safe or bps
 is currently determing if the battery pack is safe. (This is arbitrary though)
 Frequency: Only sends when tripped
 Notes: Whenever tripped, the bps will sequentially send out every CAN
-message related to the BPS from BPS trip to CAN Error. The bps will not periodically send 0 for BPS trip on the CAN line. The act of sending the BPS trip CAN ID itself should alarm all systems that it tripped"
+message related to the BPS from BPS trip to CAN Error. The bps will not periodically send 0 for BPS trip on the CAN line. The act of sending the BPS trip CAN ID itself should alarm all systems that it tripped"
 BPS All Clear,0x101,1,0,,"Precision: 1 bit
 Data: 1 - for everything is running and safe
 0 - for not clear. Could either mean the bps hasn't finished
 diagnostics on battery pack or the bps tripped
-Frequency: Only on startup"
-BPS Contactor State,0x102,1,0,,"Precision: 1 bit
-Data: 1 - contactor on
-0 - contactor off
-Frequency: Low - sent whenever it turns on or off"
+Frequency: Only on startup"
+BPS Contactor State,0x102,1,0,,"Precision: 3 bits
+Data: 
+000 - All BPS contactors are open
+011 - HV Plus/Minus BPS Contactors are open, HV Array Contactor is closed
```

Comment:
I think the 011 might be a typo- should it be 001?

Reply:
You're right -- Thanks!